docs/QuestionsIssuesPrs.md

```diff
@@ -20,7 +20,8 @@ If you don't get an answer within 1-2 days, feel free to post another comment in
 
 ## Submitting a PR
 
-> This is an adaptation of the [_GitHub Flow_](https://guides.github.com/introduction/flow/). 
+> _Ask a programmer to review 10 lines of code, he'll find 10 issues. Ask him to do 500 lines and he'll say it 
+> looks good. [[source](https://twitter.com/girayozil/status/306836785739210752)]_
```

Comment:
Hrm it seems kind of strange to have this quote directly below the "Submitting a PR" heading.

Maybe we can remove this quote for now? If we ever had a document or section dedicated specifically to good commit organization, we could add this quote back in again.